components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/util/TestUtils.java

```diff
@@ -192,33 +196,37 @@ public static String buildJWT(String issuer, String subject, String jti, String
             jwtClaimsSetBuilder.notBeforeTime(new Date(issuedTime + notBeforeMillis));
         }
         JWTClaimsSet jwtClaimsSet = jwtClaimsSetBuilder.build();
-        if (JWSAlgorithm.NONE.getName().equals(algorythm)) {
+        if (JWSAlgorithm.NONE.getName().equals(algorithm)) {
             return new PlainJWT(jwtClaimsSet).serialize();
         }
-
-        return signJWTWithRSA(jwtClaimsSet, privateKey);
+        JWSAlgorithm jwsAlgorithm = JWSAlgorithm.parse(algorithm);
+        return signJWTWithRSA(jwtClaimsSet, privateKey, jwsAlgorithm);
     }
 
     /**
      * sign JWT token from RSA algorithm
      *
      * @param jwtClaimsSet contains JWT body
      * @param privateKey
+     * @param jwsAlgorithm JWS algorithm
      * @return signed JWT token
      * @throws RequestObjectException
      */
-    public static String signJWTWithRSA(JWTClaimsSet jwtClaimsSet, Key privateKey)
+    public static String signJWTWithRSA(JWTClaimsSet jwtClaimsSet, Key privateKey, JWSAlgorithm jwsAlgorithm)
```

Comment:
Since this is a public method overload it without changing the method signature.

Reply:
Fixed in f409e32a5e8cad00a5dfeaaac027011720f2c534